test/e2e/common.go

```diff
@@ -51,6 +51,8 @@ const (
 	AzureCPSubnetCidr        = "AZURE_CP_SUBNET_CIDR"
 	AzureNodeSubnetCidr      = "AZURE_NODE_SUBNET_CIDR"
 	MultiTenancyIdentityName = "MULTI_TENANCY_IDENTITY_NAME"
+	ClusterIdentityName      = "CLUSTER_IDENTITY_NAME"
+	VMSSHPort                = "VM_SSH_PORT"
```

Comment:
are you using VMSSHPort anywhere? It was removed in another PR

Reply:
looks like the rebase messed up something, will clean up

Reply:
done